gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineFormat.java

```diff
@@ -46,14 +47,12 @@ public void apply(Project project) {
                 // use empty string to specify one group for all non-static imports
                 java.importOrder("");
                 java.trimTrailingWhitespace();
-                java.indentWithSpaces(4);
-                java.endWithNewline();
+
+                java.eclipse().configFile(
+                        project.file(Paths.get(getConfigDir(), "spotless/eclipse.xml").toString()));
 
                 // No empty lines at start of blocks
                 java.replaceRegex("Block starts with blank lines", "\\) \\{\n+", ") {\n");
```

Comment:
I think we need to delete this too, because the eclipse formatter and this regex can fight each other. In this example, when you define an anonymous class that has a field, eclipse wants to add a line and this regex tries to delete it.

```diff
    private static <T> Spliterator<T> takeWhile(Spliterator<T> delegate, Predicate<? super T> predicate) {
        int newCharacteristics = delegate.characteristics() & ~Spliterator.CONCURRENT;
        return new Spliterators.AbstractSpliterator<T>(delegate.estimateSize(), newCharacteristics) {
+
            private boolean stillGoing = true;

            @Override
            public boolean tryAdvance(Consumer<? super T> consumer) {
             //...
            }
        };
    }
```